lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/table/QueryResultBundle.json

```diff
@@ -47,6 +47,10 @@
 			"description": "The date-time when this table/view was last updated. Note: Since views are eventually consistent a view might still be out-of-date even if it was recently updated. Use mask = 0x80 to include in the bundle.",
 			"type": "string",
 			"format": "date-time"
+		},
+		"combinedSql": {
+			"description": "A SQL statement that would return the same results as the request Query object. Use mask = 0x100 to include in the bundle.",

```

Comment:
The SQL that is the combination of a the input SQL, FacetRequests, AdditionalFilters, Sorting, and Pagination.